src/components/pages/Playground/Playground.jsx

```diff
@@ -36,44 +36,16 @@ const Playground = () => {
           Learn React
         </a>
       </Typography>
-      <h3>Primary Button</h3>
-      <div className="buttonContainer buttonContainerColor">
-        <Button type="primary" onClick={() => alert("Primary Button Clicked")}>
-          Primary Button
-        </Button>
-
-        <Button type="primary" disabled>
-          Primary Button
-        </Button>
-      </div>
-
-      <h3>Secondary Button</h3>
-      <div className="buttonContainer buttonContainerColor">
-        <Button
-          type="secondary"
-          onClick={() => alert("Secondary Button Clicked")}
-        >
-          Secondary Button
-        </Button>
-
-        <Button type="secondary" disabled>
-          Secondary Button
-        </Button>
-      </div>
-
-      <h3>Tertiary Button</h3>
-      <div className="buttonContainer">
-        <Button
-          type="tertiary"
-          onClick={() => alert("Tertiary Button Clicked")}
-        >
-          Tertiary Button
-        </Button>
-        <Button type="tertiary" disabled>
-          Tertiary Button
-        </Button>
-      </div>
-
+      <Button style="white" onClick={() => alert("Primary Button Clicked")}>
+        White Color Button
+      </Button>
+      <Button style="dark" onClick={() => alert("Secondary Button Clicked")}>
+        Dark Color Button
+      </Button>
```

Comment:
Please put a light-colored background behind the Dark Color Button to make it visible. The contrast is too low to notice it on the dark background.

Reply:
agree with Joan